biogeochem/EDPhysiologyMod.F90

```diff
@@ -751,8 +751,14 @@ subroutine seeds_in( currentSite, cp_pnt )
        do while (associated(currentCohort))
           do p = 1, numpft
              if (pft_present(p)) then
-                currentPatch%seeds_in(p) = currentPatch%seeds_in(p) +  currentCohort%seed_prod * currentCohort%n / &
+	        if(EDPftvarcon_inst%woody(p)==itrue)then
+                  currentPatch%seeds_in(p) = currentPatch%seeds_in(p) +  currentCohort%seed_prod * currentCohort%n / &
                      (currentPatch%area * npfts_present)
+		else
+		  !for grasses, the bstore goes to the reproduction through colonial reproduction when grass dies
+                  currentPatch%seeds_in(p) = currentPatch%seeds_in(p) + (currentCohort%seed_prod * currentCohort%n - &
```

Comment:
do we want this to always happen, or only happen in salty mode? (I can see why we might want it to always happen...)

Reply:
@rosiealice , good point and I can add a parameter in the input file so that only a fraction of storage goes to reproduction. If the fraction set to zero by default, then no storage goes to reproduction.

Reply:
@rosiealice , I have added a new parameter in the fates parameter file so that a specified  proportion of bstore goes to reproduction. If needed, this parameter can be used for woody plants for resprout. The default is set to zero so that the code behaviors should be same as before.